.ai/auto-review/repository/github.py

```diff
@@ -23,7 +23,24 @@ def post_comment_to_line(self, text, commit_id, file_path, line):
             "body": text,
             "commit_id": commit_id,
             "path" : file_path,
-            "position" : line
+            "side":"RIGHT",
+            "line" : line
+        }
```

Comment:
28 : Incorrect usage of dictionary union. `self.__header_accept_json` and `self.__header_authorization` have the same key "Authorization", the second value will overwrite the first : Use `headers = self.__header_accept_json.copy()` then `headers.update(self.__header_authorization)`